soccer/Robot.cpp

```diff
@@ -178,8 +170,11 @@ void OurRobot::moveTuning(Geometry2d::Point goal, float endSpeed) {
              << ", " << goal.y() << ")" << endl;
     }
 
-    intent().motion_command = std::make_unique<Planning::TuningPathCommand>(
-        MotionInstant(goal, (goal - pos()).normalized() * endSpeed));
+    Geometry2d::Point targetPoint = goal;
+    Geometry2d::Point targetVel = (goal - pos()).normalized() * endSpeed;
+    RobotInstant goal_instant{Geometry2d::Pose{targetPoint, 0},
+                              Geometry2d::Twist{targetVel, 0}, RJ::Time{0s}};
+    setMotionCommand(Planning::PathTargetCommand{goal_instant});
```

Comment:
This is going from TuningPathCommand -> PathTargetCommand. I haven't looked, but I imagine they're different

Reply:
We're removing the old `TuningPathCommand` for now. I'll add a TODO for it.